translations/export.eo.xlf

```diff
@@ -0,0 +1,16 @@
+ 
```

Comment:
please remove the new line, it is invalid XML otherwise and the build fails

Reply:
Fixed. Thanks for info!